.travis.yml

```diff
@@ -22,7 +22,7 @@ env:
 
     # Configuration variables.
     - DRUPAL_TI_MODULE_NAME="akamai"
-    # - DRUPAL_TI_SIMPLETEST_GROUP="akamai"
+    - DRUPAL_TI_SIMPLETEST_GROUP="akamai"
```

Comment:
Yep. I am fiddling with travis on a branch over here if you want to have a play:  https://github.com/cam8001/akamai/tree/travis-updates
https://travis-ci.org/cam8001/akamai